Dockerfiles/agent/Dockerfile

```diff
@@ -75,25 +74,6 @@ RUN find / -maxdepth 1 -name "${DD_AGENT_ARTIFACT}" -exec tar xvf {} -C . \; \
     opt/datadog-agent/embedded/share/man \
     # self-test certificates that are detected (false positive) as private keys
     opt/datadog-agent/embedded/lib/python*/site-packages/future/backports/test \
- && if [ "$PYTHON_VERSION" = "2" ]; then \
-        rm -rf \
-            opt/datadog-agent/embedded/bin/2to3-3* \
-            opt/datadog-agent/embedded/bin/easy_install-3* \
-            opt/datadog-agent/embedded/bin/idle* \
-            opt/datadog-agent/embedded/bin/pip3* \
-            opt/datadog-agent/embedded/bin/pydoc* \
-            opt/datadog-agent/embedded/bin/python3* \
-            opt/datadog-agent/embedded/bin/pyvenv* \
-            opt/datadog-agent/embedded/include/python3* \
-            opt/datadog-agent/embedded/lib/*python3* || true ;\
-    fi \
- && if [ "$PYTHON_VERSION" = "3" ]; then \
-        rm -rf \
-            opt/datadog-agent/embedded/bin/pip2* \
-            opt/datadog-agent/embedded/bin/python2* \
-            opt/datadog-agent/embedded/include/python2* \
-            opt/datadog-agent/embedded/lib/*python2* || true ;\
-    fi \
```

Comment:
I assume we no longer need to run this command because we no longer install Python, right?